piksi_tools/console/update_downloader.py

```diff
@@ -23,19 +23,29 @@ def __init__(self):
     self.index = jsonload(f)
     f.close()
 
-  def download_stm_firmware(self):
+  def download_stm_firmware(self, hwrev):
     try:
-      url = self.index['piksi_v2.3.1']['stm_fw']['url']
+      url = self.index[hwrev]['stm_fw']['url']
```

Comment:
This means we will add the STM_FW key into the index file.  I find this a little "wrong" from first principles because there is no "stm" any longer. 

Can we add the switch statement on hwrev here, or should we make the index shema conform?